packages/kbn-es/src/utils/docker.test.ts

```diff
@@ -556,8 +556,7 @@ describe('resolveDockerCmd()', () => {
 describe('runDockerContainer()', () => {
   test('should resolve', async () => {
     execa.mockImplementation(() => Promise.resolve({ stdout: '' }));
-
-    await expect(runDockerContainer(log, {})).resolves.toEqual({ stdout: '' });
+    await expect(runDockerContainer(log, {})).resolves.toBeUndefined();
```

Comment:
Related to this change https://github.com/elastic/kibana/blob/4ccd77230a18c52b59bd8e18d2cc36d73baab231/packages/kbn-es/src/utils/docker.ts#L717